### PR TITLE
Add reference to the JSON schema.  Resolves issue #270

### PR DESCRIPTION
**Pull request by goneall**
Note that the hyperlink in the doc will not work until the development branch is merged into master.

Signed-off-by: Gary O'Neall <gary@sourceauditor.com>

### CODE REVIEW

File: chapters/1-rationale.md [hunk 1]
```diff
@@ -74,6 +74,7 @@ In an SPDX document, Relationship elements can be used to indicate relationships
 * YAML 1.2  see: https://yaml.org/spec/1.2/spec.html
 * JavaScript Object Notation (JSON) see: ECMA-404(https://www.ecma-international.org/publications/files/ECMA-ST/ECMA-404.pdf)
 * Resource Description Framework (RDF also referred to as RDF/XML) see: https://www.w3.org/TR/rdf-syntax-grammar/
+    The JSON Schema for SPDX can be found in the [SPDX Spec Git Repository Schema directory](https://github.com/spdx/spdx-spec/schemas/spdx-schema.json)
```
Comment (swinslow) [inline chapters/1-rationale.md:77]: Hi @goneall, content-wise this looks good, but putting it here looks like it's getting folded into the RDF bullet point when it renders. Can this maybe be moved one line up and made into a sub-bullet?

**Comment by carmenbianca** [comment]
@kestewart The URL in this PR leads to a 404. I assume it needs to lead to https://github.com/spdx/spdx-spec/blob/development/v2.2/schemas/spdx-schema.json. `blob/devepment/v2.2` should probably be replaced by `blob/master`, although `master` doesn't currently have this file yet.

**Comment by kestewart** [comment]
Thanks for flagging @carmenbianca.   @goneall -  where do you want it to reside long terms so we can clean this up?

**Comment by goneall** [comment]
This should be resolved once this is merged into master.  If we pointed it to 2.2, it would create a 404 after the development branch is deleted.

**Comment by goneall** [comment]
@carmenbianca @kestewart I just checked and indeed the link was still broken after merging to master.  I just created a PR to fix.

Thanks @carmenbianca for pointing this out.

**Pull request merged**